gnome-selected-20170316.json

```diff
@@ -0,0 +1,16 @@
+{
+  "tags": [
+    "ENV-DISTRI-opensuse",
+    "gnome-selected"
+  ],
+  "area": [
+    {
+      "width": 162,
+      "type": "match",
+      "xpos": 358,
+      "ypos": 373,
+      "height": 12
+    }
+  ],
+  "properties": []
+}
```

Comment:
well, release team decide to take it up first if the function is worked. all discussion about UI improvement can be continue, but that is NOT the reason blocked this, and not a reason rejecting it from openqa POV.

and superseding request is catching us.